metrics_utility/management/commands/build_report.py

```diff
@@ -124,7 +124,17 @@ def handle(self, *args, **options):
             extra_params=extra_params).create()
 
         if report_dataframe[0] is None or report_dataframe[0].empty:
-            self.logger.info(f"No billing data for month: {opt_month}")
+            if (opt_since is not None or opt_until is not None):
+                log = 'No billing data for date range: '
```

Comment:
use the extra_params['since_date'] here
```
f"No billing data for date range: since={extra_params['since_date']} until={extra_params['until_date']}"
```